Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.0
+FROM ruby:3.4
```

Comment:
should this use either images from dockerhub-images (our mirror) or docker-images-base?

old version of it ran into ratelimit [here](https://github.com/zendesk/techmenu/actions/runs/17123180673/job/48568646959#step:3:11)